components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/cmd/SetupCmd.java

```diff
@@ -432,4 +426,34 @@ private void populateHosts(String host) {
             throw new CLIInternalException("Error occurred while setting up URL configurations.");
         }
     }
+
+    private static void init(String projectName, String configPath, String deploymentConfigPath) {
+        try {
+            GatewayCmdUtils.createProjectStructure(projectName);
+            GatewayCmdUtils.createDeploymentConfig(projectName, deploymentConfigPath);
+
+            Path configurationFile = Paths.get(configPath);
+            if (Files.exists(configurationFile)) {
+                Config config = TOMLConfigParser.parse(configPath, Config.class);
+                GatewayCmdUtils.setConfig(config);
+            } else {
+                logger.error("Configuration: {} Not found.", configPath);
+                throw new CLIInternalException("Error occurred while loading configurations.");
+            }
+
+            deploymentConfigPath = GatewayCmdUtils.getDeploymentConfigLocation(projectName);
+            ContainerConfig containerConfig = TOMLConfigParser.parse(deploymentConfigPath, ContainerConfig.class);
+            GatewayCmdUtils.setContainerConfig(containerConfig);
+
+            CodeGenerationContext codeGenerationContext = new CodeGenerationContext();
+            codeGenerationContext.setProjectName(projectName);
+            GatewayCmdUtils.setCodeGenerationContext(codeGenerationContext);
+        } catch (ConfigParserException e) {
+            logger.error("Error occurred while parsing the configurations {}", configPath, e);
+            throw new CLIInternalException("Error occurred while loading configurations.");
+        } catch (IOException e) {
+            logger.error("Error occurred while generating project configurationss", e);
+            throw new CLIInternalException("Error occurred while loading configurations.");
+        }
+    }
```

Comment:
Looks like you have not done any changes to the init method and only change the position of the method. Shall we revert the change unless you have a valid reason to do this change?